src/types/input.ts

```diff
@@ -10,3 +10,27 @@ export type InputBaseProps = {
     onChange?: (value: string) => void;
     'data-test-id'?: string;
 } & Omit<InputHTMLAttributes<HTMLInputElement>, 'className' | 'onChange'>;
+
+export type InputTypes =
```

Comment:
```suggestion
export type InputType =
```
Always singular for types